xcorim.cddl

```diff
@@ -1,65 +1,47 @@
-xcorim = 
-  #6.47113(unsigned-xcorim) / 
+corim-revocation =
+  #6.47113(unsigned-xcorim-map) /
```

Comment:
As per the document, Line 3 is 
signed-xcorim-map

Reply:
I stop and pondered on that, but it's not a map :-)

Reply:
I modeled it as a map with a single member. 
xcorim.cose-signature = 0
Value Type is '#6.18(signed-xcorim-map)' which is a group type i think.
I'm not passionate about it, but seems to fit the table format.

Reply:
I think I'd rather avoid the cognitive dissonance of using -map for this.